src/Database/Redis/Connection.hs

```diff
@@ -117,6 +128,9 @@ defaultConnectInfo = ConnInfo
     , connectTLSParams      = Nothing
     }
 
+shardMapRefreshDelta :: Time.NominalDiffTime
+shardMapRefreshDelta = Time.secondsToNominalDiffTime 60
+
```

Comment:
Where is this being used?

Reply:
@neeraj97 this was used for a change that I added briefly for fixing the dead lock which was caused due to refreshShardMap function. Whenever a refresh was called by some thread, no other call to refreshShardMap would do the refresh until `shardMapRefreshDelta` time has passed. I shifted to using a combination of `tryTakeMVar` and the blocking nature of `readMVar` to achive this but then forgot to remove this. Not sure why haskell compiler didn't give unused warning even though this is not being used anywhere

Reply:
I have removed this now